### ISSUE TITLE
Can't open KDiff3

### ISSUE THREAD

**Issue by GoogleCodeExporter** [salvatorelai/svnx#162]
```
What steps will reproduce the problem?
1. Go to preferences.
2. Change diff program to KDiff3.
3. Try to diff a modified file.

What is the expected output? What do you see instead?
Expected: Open KDiff3 with files preloaded in there.
Current behaviour: KDiff3 asks to enter path to files to diff them.

What version of the product are you using? On what operating system?
svnX: 1.3.2
KDiff3: 0.9.95

Please provide any additional information below.


```

Original issue reported on code.google.com by `tol...@gmail.com` on 11 Mar 2012 at 10:13

**Comment by GoogleCodeExporter** [comment]
```
Assuming you have a symbolic link in ~/bin to the kdiff3.app’s binary (see 
svnX’s Help — “3.11. Diff”).
Please try changing the following line in 
svnX.app/Contents/Resources/svndiff.sh from:
    "kdiff3"        ) open -a kdiff3 "$file1" "$file2" &> /dev/null ;;
to:
    "kdiff3"        ) ~/bin/kdiff3 "$file1" "$file2" &> /dev/null ;;

Does that fix the problem?
```

Original comment by `chris...@gmail.com` on 12 Mar 2012 at 12:53

**Comment by GoogleCodeExporter** [comment]
```
Yes, this fixed the problem. Thanks.
```

Original comment by `tol...@gmail.com` on 12 Mar 2012 at 12:59

**Comment by GoogleCodeExporter** [comment]
```
Fixed in svnX 1.3.3.
```

Original comment by `chris...@gmail.com` on 9 Apr 2012 at 11:08
- Changed state: **Fixed**
